pymc/logprob/order.py

```diff
@@ -102,7 +107,15 @@ def find_measurable_max(fgraph: FunctionGraph, node: Node) -> Optional[List[Tens
     if axis != base_var_dims:
         return None
 
-    measurable_max = MeasurableMax(list(axis))
+    # distinguish measurable discrete and continuous (because logprob is different)
+    if base_var.owner.op.dtype.startswith("int"):
```

Comment:
Maybe use [`pymc.vartypes.discrete_types`](https://github.com/pymc-devs/pymc/blob/fafe89a9617db3be9ac8e82a48a1678fb046ddc9/pymc/vartypes.py#L32C1-L32C15) here?

Reply:
Yes. Once the meta info PR is ready, we will explicitly check for the measure type here itself.